Admin/Admin.php

```diff
@@ -2030,12 +2052,15 @@ public function getFilterFieldDescriptions()
     /**
      * {@inheritdoc}
      */
-    public function addChild(AdminInterface $child)
+    public function addChild(AdminInterface $child, $field = null)
     {
         $this->children[$child->getCode()] = $child;
 
         $child->setBaseCodeRoute($this->getCode().'|'.$child->getCode());
         $child->setParent($this);
```

Comment:
I will check everything on Monday and push a new commit with all you comment.

I must say we didn't get any error or problem using this since I made the PR. Even if with single parents or empty parents. But I will check ;)

Enviado desde mi iPhone

> El 9 abr 2016, a las 18:13, Grégoire Paris <notifications@github.com> escribió:
> 
> In Admin/Admin.php:
> 
> > @@ -2030,12 +2052,15 @@ public function getFilterFieldDescriptions()
> >      /**
> >       * {@inheritdoc}
> >       */
> > -    public function addChild(AdminInterface $child)
> > +    public function addChild(AdminInterface $child, $field = null)
> >      {
> >          $this->children[$child->getCode()] = $child;
> >  
> >          $child->setBaseCodeRoute($this->getCode().'|'.$child->getCode());
> >          $child->setParent($this);
> This is just what I put in my PR description.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>